### PR TITLE
fix: changing video providers

### PR DESCRIPTION
**Pull request by GeronimoJohn**
# Description

Changing video providers is broken on admin. Specifically changing to an internal video break the admin or gives out an error

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/29494894/todos/5373621635)

# How should this PR be QA Tested?

Create a video block and select a video
- [x] Changing video providers of a video block should work
- [x] Changing video providers of a background video should work

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged into main

### CODE REVIEW

**Review by tataihono — APPROVED** [review]

**Review by tataihono — APPROVED** [review]

**Review by tataihono — DISMISSED** [review]
missed failing test

**Review by tataihono — DISMISSED** [review]

**Comment by jianwei1** [comment]
> # How should this PR be QA Tested?
> Create a video block and select a video
> 
> * [x]  Changing video providers of a video block should work
> * [x]  Changing video providers of a background video should work
> 
@GeronimoJohn checks pass on desktop

File: apps/journeys/next.config.js [hunk 1]
```diff
@@ -13,7 +13,9 @@ const nextConfig = {
       'images.unsplash.com',
       'localhost',
       'unsplash.com',
-      'imagizer.imageshack.com'
+      'imagizer.imageshack.com',
+      'd1wl257kev7hsz.cloudfront.net',
```
Comment (tataihono) [inline apps/journeys/next.config.js:17]: can you put a comment next to this indicating that its for arclight content?

**Review by tataihono — DISMISSED** [review]

**Review by tataihono — APPROVED** [review]

File: libs/journeys/ui/src/components/Card/ContainedCover/ContainedCover.tsx [hunk 2]
```diff
@@ -73,6 +73,8 @@ export function ContainedCover({
     }
   }, [imageBlock, theme, videoBlock, backgroundBlur])
 
+  const videoImage = videoBlock?.image ?? videoBlock?.video?.image
```
Comment (tataihono) [inline libs/journeys/ui/src/components/Card/ContainedCover/ContainedCover.tsx:76]: Wonder if this is worth making deliberate.

```suggestion
  const videoImage = videoBlock?.source === VideoBlockSource.internal ? videoBlock?.video?.image : videoBlock?.image
```

Comment (tataihono) [reply libs/journeys/ui/src/components/Card/ContainedCover/ContainedCover.tsx:76]: same also for the video component.

**Review by tataihono — CHANGES_REQUESTED** [review]

**Comment by tataihono** [comment]
<div style="position: relative; padding-bottom: 62.5%; height: 0;"><iframe src="https://www.loom.com/embed/0f97caeaa25a477fbdc14d87995087ab" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>

I was able to QA this successfully.

**Review by tataihono — APPROVED** [review]

**Pull request merged**